middleware/classify/util.go

```diff
@@ -1,8 +1,11 @@
 package classify
 
-// IndexAliasCache cache to store index alias map
+// IndexAliasCache cache to store index -> alias map
 var IndexAliasCache = make(map[string]string)
 
+// AliasIndexCache cache to store alias -> index map
+var AliasIndexCache = make(map[string]string)
```

Comment:
@lakhansamani What's the need to manage two caches, you can retrieve the `alias to index` map from the old one.

Reply:
In order to do that I would have to loop over whole map. This is solution is more efficient.